models/business_models/product/newtab/newtab.yml

```diff
@@ -0,0 +1,5 @@
+version: 2
+
+models:
+  - name: newtab_visits
```

Comment:
I noticed the .sql file is called `newtab_visits_v1.sql`. Should this YML file also refer to `newtab_visits_v1` instead of `newtab_visits`?

We can either manually version our models, or use dbt's built-in model versioning mechanism:
https://docs.getdbt.com/docs/collaborate/govern/model-versions#how-is-this-different-from-just-creating-a-new-model

Reply:
I think having the version in the filename is required to use dbt's built in versioning mechanism. Based on the docs _dbt expects each model version to be defined in a file named `<model_name>_v<v>`_. But yes, we should definitely follow the versioning that dbt provides.

Reply:
You are totally right. In that case, the current set of files (and file names) makes sense.

I wonder if we should:
- get rid of `version: 2` in the YML file (dbt says this is "optional starting from dbt v1.5" and it's a confusing reference to the schema version of the YML file itself)
- include `latest_version: 1` below the `name` field in the newtab.yml file

That way, I think someone in the codebase could reasonably deduce that the `*_v1.sql` file is described by the `*.yml` file and that they can create a `*_v2.sql` file for any breaking diffs. How does that sound?